docs/dictionary/wordlist.txt

```diff
@@ -650,9 +658,9 @@ subjectsschema
 submissionid
 submissionslice
 submissiontype
+submitter's
 submitterdemultiplexed
 submitterid
-submitter's
```

Comment:
🤨

Reply:
well yeah smth is off in the sorting, will revert that

Reply:
If you keep it just like that, the sorting on my end will at least never change it again :D

Reply:
i think there is a difference between how vscode sorts and how the pre-commit hook `sort.sh` sorts